<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>453</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>404</width>
    <height>453</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>404</width>
    <height>453</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Rompe Bolas</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../recursos/recursos.qrc" >:/imagenes/icono.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="LPanel" native="1" name="widget" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>50</y>
      <width>351</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="puntos_label" >
       <property name="text" >
        <string>Puntuación</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="puntos" >
       <property name="text" >
        <string>0</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="puntosJugadaActual" >
    <property name="geometry" >
     <rect>
      <x>300</x>
      <y>400</y>
      <width>54</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>404</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuJuego" >
    <property name="title" >
     <string>&amp;Juego</string>
    </property>
    <addaction name="actionNuevo" />
    <addaction name="actionPreferencias" />
    <addaction name="separator" />
    <addaction name="actionSalir" />
   </widget>
   <widget class="QMenu" name="menuAyuda" >
    <property name="title" >
     <string>&amp;Ayuda</string>
    </property>
    <addaction name="action_C_mo_se_juega" />
   </widget>
   <addaction name="menuJuego" />
   <addaction name="menuAyuda" />
  </widget>
  <action name="actionNuevo" >
   <property name="icon" >
    <iconset resource="../recursos/recursos.qrc" >:/imagenes/nuevopj.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Nueva partida</string>
   </property>
   <property name="iconText" >
    <string>Nueva Partida</string>
   </property>
   <property name="toolTip" >
    <string>Empieza una nueva partida</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSalir" >
   <property name="icon" >
    <iconset resource="../recursos/recursos.qrc" >:/imagenes/apagar.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Salir</string>
   </property>
   <property name="toolTip" >
    <string>Sale del programa</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_C_mo_se_juega" >
   <property name="text" >
    <string>¿&amp;Cómo se juega?</string>
   </property>
  </action>
  <action name="actionPreferencias" >
   <property name="icon" >
    <iconset resource="../recursos/recursos.qrc" >:/imagenes/preferencias.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Preferencias...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+P</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LPanel</class>
   <extends>QWidget</extends>
   <header>lpanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../recursos/recursos.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>actionSalir</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>249</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
